tests/foreman/api/test_organization.py

```diff
@@ -349,9 +349,9 @@ def test_positive_add_smart_proxy(self):
         @CaseLevel: Integration
         """
         # Every Satellite has a built-in smart proxy, so let's find it
-        smart_proxies = entities.SmartProxy().search()
-        self.assertGreater(len(smart_proxies), 0)
```

Comment:
Is there any reason to remove this verification? Does it mean that you absolutely sure that default capsule exist?